.travis.yml

```diff
@@ -195,14 +194,22 @@ jobs:
             - ubuntu-toolchain-r-test
           packages:
             - g++-5
+            - valgrind
       before_install:
         - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
       install:
-        - ccache -z
-        - ccache --max-size=1G
-        - cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Release'  '-DCMAKE_CXX_COMPILER=/usr/bin/g++-5'
+        - cmake -H. -Bbuild '-DCMAKE_BUILD_TYPE=Debug'  '-DCMAKE_CXX_COMPILER=/usr/bin/g++-5'
```

Comment:
Do you want/need Debug here, or would RelWithDebInfo be a closer representative of the memory performance of a release build?

Reply:
Probably true - will use that!